main.go

```diff
@@ -86,13 +67,16 @@ func main() {
 		if err != nil {
 			logger.Fatal(err)
 		}
-		pprof.StartCPUProfile(f)
+		err = pprof.StartCPUProfile(f)
+		if err != nil {
```

Comment:
Shorter alternative (here and elsewhere):

```suggestion
		if err := pprof.StartCPUProfile(f); err != nil {
```

Reply:
:+1: